src/main/java/io/stargate/sgv2/jsonapi/service/operation/builder/BuiltConditionPredicate.java

```diff
@@ -10,15 +10,16 @@ public enum BuiltConditionPredicate {
   IN("IN"),
   CONTAINS("CONTAINS"),
   NOT_CONTAINS("NOT CONTAINS"),
-  CONTAINS_KEY("CONTAINS KEY"),
-  ;
+  CONTAINS_KEY("CONTAINS KEY");
 
-  private final String cql;
+  public final String cql;
 
   BuiltConditionPredicate(String cql) {
     this.cql = cql;
   }
 
+  // TIDY - remove this use of toString() it should be used for log msg's etc, not core
+  // functionality. This is called to build the CQL string we execute.
```

Comment:
Agreed, `toString()` is for diagnostics etc, not for "real" use cases.

I have no strong opinions b/w opening up public String field (as done here) or separate access method.